### PR TITLE
refactor: Update remaining cell-counting parsers to schema mapper

### PR DESCRIPTION
**Pull request by nathan-stender**

### CODE REVIEW

File: tests/parsers/roche_cedex_hires/testdata/roche_cedex_hires_example_3.json [hunk 1]
```diff
@@ -2386,17 +2386,41 @@
                                             "value": 0.01331124444,
                                             "unit": "10^6 cells/mL"
                                         },
+                                        "average live cell diameter (cell counter)": {
```
Comment (nathan-stender) [inline tests/parsers/roche_cedex_hires/testdata/roche_cedex_hires_example_3.json:2389]: Note to reviewer: this was a bug in original implementation, turns out walrus operator evaluates 0 as falsey!!!

File: tests/parsers/roche_cedex_hires/testdata/roche_cedex_hires_example_3.json [hunk 2]
```diff
@@ -2386,17 +2386,41 @@
...
+                                            "value": 0.0,
                                             "unit": "cell"
                                         },
                                         "dead cell count": {
-                                            "value": 13,
+                                            "value": 13.0,
                                             "unit": "cell"
                                         },
                                         "custom information document": {
+                                            "average compactness": {
```
Comment (nathan-stender) [inline tests/parsers/roche_cedex_hires/testdata/roche_cedex_hires_example_3.json:2406]: Ditto these

File: src/allotropy/parsers/roche_cedex_hires/roche_cedex_hires_reader.py [hunk 3]
```diff
@@ -34,13 +34,17 @@ def read(cls, named_file_contents: NamedFileContents) -> pd.DataFrame:
...
             )
         elif named_file_contents.original_file_name.endswith(".xlsx"):
-            return read_excel(named_file_contents.contents.name)
+            df = read_excel(named_file_contents.contents.name)
         else:
             message = f"{constants.UNSUPPORTED_FILE_FORMAT_ERROR} '{named_file_contents.original_file_name}'"
             raise AllotropeConversionError(message)
+
+        # Fix typo found in some source files.
+        df.columns = df.columns.str.replace("identifer", "identifier", regex=True)
```
Comment (mdlu) [inline src/allotropy/parsers/roche_cedex_hires/roche_cedex_hires_reader.py:49]: does this happen in real life ???

File: src/allotropy/parsers/chemometec_nucleoview/nucleoview_structure.py [hunk 4]
```diff
@@ -1,66 +1,77 @@
...
+    return MeasurementGroup(
+        analyst=data.get(str, "Operator", DEFAULT_ANALYST),
+        measurements=[
+            Measurement(
+                measurement_identifier=random_uuid_str(),
+                timestamp=timestamp or DEFAULT_EPOCH_TIMESTAMP,
+                sample_identifier=data[str, "Image"].split("-")[3],
+                cell_density_dilution_factor=data.get(float, "Multiplication factor"),
+                viability=data[float, "Viability (%)"],
+                # Cell counts are measured in cells/mL, but reported in millions of cells/mL
```
Comment (mdlu) [inline src/allotropy/parsers/chemometec_nucleoview/nucleoview_structure.py:57]: I see this is happening a lot ... is it worth putting a helper that automatically divides? up to you

File: tests/parsers/roche_cedex_hires/testdata/roche_cedex_hires_example_1.json [hunk 5]
```diff
@@ -51,23 +51,23 @@
                                             "unit": "10^6 cells/mL"
                                         },
                                         "dead cell density (cell counter)": {
-                                            "value": 0.2237332344,
+                                            "value": 0.22373323439999998,
```
Comment (mdlu) [inline tests/parsers/roche_cedex_hires/testdata/roche_cedex_hires_example_1.json:54]: is this expected? love floating point 🤣 

mostly wondering how did this happen? seems to be happening pretty regularly (when it wasn't before) this is where we divide by `1e6` right? is there a way to preserve sigfigs or something

File: tests/parsers/roche_cedex_hires/testdata/roche_cedex_hires_example_2.json [hunk 6]
```diff
@@ -51,23 +51,23 @@
                                             "unit": "10^6 cells/mL"
                                         },
                                         "dead cell density (cell counter)": {
-                                            "value": 0.2237332344,
+                                            "value": 0.22373323439999998,
```
Comment (mdlu) [inline tests/parsers/roche_cedex_hires/testdata/roche_cedex_hires_example_2.json:54]: same here (and a couple other instances)

**Review by mdlu — CHANGES_REQUESTED** [review]

File: src/allotropy/parsers/roche_cedex_hires/roche_cedex_hires_reader.py (hunk 3 above)
Comment (nathan-stender) [reply src/allotropy/parsers/roche_cedex_hires/roche_cedex_hires_reader.py:49]: Sadly yes 🤣

File: tests/parsers/roche_cedex_hires/testdata/roche_cedex_hires_example_1.json (hunk 5 above)
Comment (nathan-stender) [reply tests/parsers/roche_cedex_hires/testdata/roche_cedex_hires_example_1.json]: Just for you, I'll make the code less pretty to preserve the values :P

**Review by stephenworlow — DISMISSED** [review]

**Review by stephenworlow — APPROVED** [review]

**Pull request merged**